cdap-app-fabric/src/main/java/co/cask/cdap/app/store/Store.java

```diff
@@ -146,6 +207,20 @@ void setStartAndRun(ProgramId id, String pid, long startTime, long runTime,
   Map<ProgramRunId, RunRecordMeta> getRuns(Set<ProgramRunId> programRunIds);
 
   /**
+   * Fetches the active (i.e STARTING or RUNNING or SUSPENDED) run records against a given NamespaceId.
+   * @param namespaceId the namespace id to match against
+   * @return map of logged runs
+   */
+  Map<ProgramRunId, RunRecordMeta> getActiveRuns(NamespaceId namespaceId);
```

Comment:
The reason I introduced these methods was to abstract out what makes a program "active" - I didn't want the user of the method to remember that I have to look for STARTING, RUNNING, and SUSPENDED program runs to constitute all active runs.

These methods will actually be eventually used in the `DistributedRunRecordCorrectorService` as well eventually.